### DISCUSSION TITLE
Add Remaining Battery Level Icon on Display With UPS Hat for Pi 5 and Pi Zero 2W.

### DISCUSSION THREAD

**Discussion by keeratsawhney** [outdoorbits/little-backup-box#339] [Ideas]
I have built a LBB using Pi5 and UPS hat to make it easy to travel and backup. Please add support for it to display Battery Level. ![20241210_173109.jpg](https://github.com/user-attachments/assets/cb6c97a0-3a42-42e2-9701-453a24dcbe73)

**Comment by keeratsawhney** [comment]
@outdoorbits  https://www.waveshare.com/wiki/UPS_HAT_(D)

**Reply by outdoorbits** [reply]
I don't understand, what do you want me to do?

**Reply by keeratsawhney** [reply]
Can you add support for Battery Level and Percentage like Battery On top of supported all display with the LBB Screen as we have currently.

**Reply by outdoorbits** [reply]
I don't use this device. Did you try the python script? I don't have too much time these days but if you could contribute testing, it might be worth to try it.
